tests/server/utilities/test_server.py

```diff
@@ -206,16 +211,11 @@ def foo(
     app.include_router(router)
 
     client = TestClient(app)
-    if packaging.version.parse(fastapi.__version__) < packaging.version.parse("0.74.0"):
```

Comment:
We own our version of FastAPI now so this doesn't apply, and when we come back with `pydantic>2` support, it will have to be on a version somewhere north of `0.100`.